API/LudumDare51.API/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,28 @@
+using LudumDare51.API.Models.Request;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LudumDare51.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class PlayerController : ControllerBase
+    {
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok();
+        }
+        
+        [HttpGet]
+        public IActionResult Get(string id)
```

Comment:
[HttpGet] should be [HttpGet("{id}")]
Add the [FromRoute] attribute to the id parameter